js_intro_and_control_flow.js

```diff
@@ -1,236 +1,290 @@
+// 7.) I have a list of favorite foods below.
+// If `Popcorn` is my favorite food and `Potato chips` my second favorite, then how would you find the rank of another food.
+// Hint: go to MDN to find an array method for finding the index of something in the array.
 
 var foods = [
-              'Popcorn',
-              'Potato chips',
-              'Shrimp',
-              'Chicken rice',
-              'Poutine',
-              'Tacos',
-              'Toast',
-              'French Toast',
-              'Crab',
-              'Pho',
-              'Lasagna',
-              'Brownie',
-              'Lobster',
-              'Donuts',
-              'Ice cream',
-              'Hamburger',
-              'Sushi',
-              'Chocolate',
-              'Pizza'
-            ];
-
-
-
-
-
-
-
-
-
-
-8.) I made a mistake with my favorite foods. How can I find the index of `Donuts` and remove it? **(Hint: look up `splice`)**
-
-
-
-
-
-
-
-
-
-9.) My friends want to know what my `5`th to `10`th favorite foods are.
-
-
-
-
-
-
-
-
-
-
-10.) How would you create an array that efficiently stored the following information:
-
-  'Moe' is 18
-  'Larry' is 19
-  'Curly' is 20
-  'Jane' is 20
-  'Emma' is 21
-  'Elizabeth' is 18
-  'Elinor' is 23
-  'Mary' is 25
-  'Darcy' is 24
-  'Grey' is 18
-  'Lydia' is 24
-  'Harriet' is 18
-
-
-
-
-
-##Javascript Control Flow Exercises
-
-1.) Nightclub Bouncer
-
-Using an if/else expression, create a script that prompts the user for their age, then:
-
-  * If the user is older than 18, he/she should receive a message that he/she may enter.
-  * If the user is less than 18, then he/she can't enter and should have a message telling him/her that he/she's too young.
-  * If the user is between 18 and 21, he/she should receive a message that he/she can enter, but not drink.
-  * If the user is older than 21, he/she should receive a message that he/she can both enter and drink.
-
-
-
-
-
-
-
-
-
-
-
-
-2.) Grade.js
-Output the following letter grade from a variable with with a test score. 
-
-Based on the user input, display either "A", "B", "C", "D", or "F", for an score that is an integer between 0 and 100. Try and use a `switch` statement.
+    'Popcorn',
+    'Potato chips',
+    'Shrimp',
+    'Chicken rice',
+    'Poutine',
+    'Tacos',
+    'Toast',
+    'French Toast',
+    'Crab',
+    'Pho',
+    'Lasagna',
+    'Brownie',
+    'Lobster',
+    'Donuts',
+    'Ice cream',
+    'Hamburger',
+    'Sushi',
+    'Chocolate',
+    'Pizza'
+];
+
+foods.indexOf();
+
+// 8.) I made a mistake with my favorite foods.How can I find the index of `Donuts` and remove it ? * * (Hint: look up `splice`) * *
+
+foods.splice(foods.indexOf('Donuts'), 1);
+
+// 9.) My friends want to know what my `5`th to `10`th favorite foods are.
+
+var whyDoMyFriendsWantToKnowThis = foods.slice(5, 11);
+
+// 10.) How would you create an array that efficiently stored the following : or,mation:
+: ',Moe'
```

Comment:
You'd probably want to create an array of objects here, with `age` and `name` keys.